board/drivers/llcan.h

```diff
@@ -12,7 +12,6 @@ void puts(const char *a);
 
 bool llcan_set_speed(CAN_TypeDef *CAN_obj, uint32_t speed, bool loopback, bool silent) {
   // initialization mode
-  CAN_obj->MCR = CAN_MCR_TTCM | CAN_MCR_INRQ;
```

Comment:
Why was this removed?

Reply:
cppcheck complains about: `Variable 'CAN_obj->MCR' is reassigned a value before the old one has been used.`

Reply:
See: https://github.com/commaai/panda/pull/243/files#diff-b3438395a72aa43adef2658b43dc64a8L32